src/tools/compiletest/src/header/tests.rs

```diff
@@ -446,15 +446,32 @@ fn wasm_special() {
         ("wasm32-unknown-emscripten", "emscripten", true),
         ("wasm32-unknown-emscripten", "wasm32", true),
         ("wasm32-unknown-emscripten", "wasm32-bare", false),
-        ("wasm32-wasi-preview1", "emscripten", false),
-        ("wasm32-wasi-preview1", "wasm32", true),
-        ("wasm32-wasi-preview1", "wasm32-bare", false),
-        ("wasm32-wasi-preview1", "wasi", true),
         ("wasm64-unknown-unknown", "emscripten", false),
         ("wasm64-unknown-unknown", "wasm32", false),
         ("wasm64-unknown-unknown", "wasm32-bare", false),
         ("wasm64-unknown-unknown", "wasm64", true),
     ];
+
+    // FIXME(yosh): We're updating the "wasm32-wasi" target name to "wasm32-wasi-preview1".
+    // The MinGW tests run a beta compiler on stage 0 which means we're having a target mismatch
+    // for a brief period. When a new beta release is cut, the `if` part of this conditional should be
+    // removed, and the `else` part should be re-inlined in the original `ignores` list.
```

Comment:
```suggestion
    // removed, and the `else` part should be re-inlined in the original `ignores` list.
    // cfg(bootstrap)
```
this tells the person updating the beta compiler that this should be removed